Source/Core/Core/IOS/SDIO/SDIOSlot0.h

```diff
@@ -136,6 +146,8 @@ class SDIOSlot0 : public Device
   std::unique_ptr<Event> m_event;
 
   u32 m_Status = CARD_NOT_EXIST;
+  u32 m_Protocol = PROTOCOL_V1;
```

Comment:
I could definitely do that, but then the member capitalization will be inconsistent.
In the light of readability, I tend to go for consistency over style rules.
Maybe what we need is just a style pass to enforce the style rules.